_layouts/home.html

```diff
@@ -6,9 +6,13 @@
 <div class="full-width-banner"></div>
 
 <div class="homepage-intro">
-    <div>
+    <div class="grid-y">
       <h1>An evolving collection of design patterns for sharing data</h1>
       <a href="/about" class="cta">What are design patterns?</a>
+      <noscript class="grid-y grid-padding-y">
```

Comment:
Including this in a `noscript` tag rather some other technique makes this very readable at a glance. Good choice!